### DISCUSSION TITLE
TSL 5.0 preview and pgm on tally.

### DISCUSSION THREAD

**Discussion by silentonset** [josephdadams/TallyArbiter#311] [Q&A] [answered]
Hi, i am new to TA. I've successfully trigger the tally of a panasonic UCU600 CCU via TSL 5.0 UDP from an ATEM CONSTELLATION.

However, whenever i put the same source on preview. It would clear the red tally even though it is still on the program bus.

Am i doing anything wrong? I've setup 4 actions for the device
- program on (red tally)
- program off (tally off)
- Preview on (green tally)
- Preview off (tally off)

Any help would be much appreciated.

**Comment by josephdadams** [comment, marked as answer]
See #16 . The TSL messages are being sent telling the device "it is in preview only", etc. For TSL devices, this is why I created the "TSL Clients" section, where you can send TSL data on any state change. See #211 , we will eventually add TSL5.0 support for the TSL clients section which will help your issue.

**Reply by silentonset** [reply]
Hi Joseph, thanks for the reply. Can i trouble you to explain how the TSL client would help my situation? Do i just add the camera as a client? I have read thru the 2 reference you gave #16 & #211 but i am still not able to understand how it works.

**Reply by josephdadams** [reply]
Yes. Once tsl5 is supported within the “tsl clients” section, you would add the IP of your camera along with the port and transport type (tcp or udp). Then all of the tally data for the entire system will be sent to these tsl clients. It will represent the entire tally state of each camera. All you have to do is make sure the Tally Arbiter Device’s “tsl address” field matches the address that your Panasonic is looking for. It should also be unique from any other devices.

This feature works great but currently only for tsl 3.1.

**Comment by silentonset** [comment]
Thanks Joseph,

Understood. If i understand this correctly. The tsl client will continuously send the last state of the device so that it is reflecting all states in correctly.

Meaning if i put my camera on pgm and and prv. The state reflected on the camera will be both red and green tally lighting up at the same time?

Looking forward to the tsl5 implementation soon.
I May try the beta 3 to see if it works:)

**Reply by josephdadams** [reply]
It depends on the implementation at the camera end.

**Comment by Erik98146** [comment]
I am also trying to get tally to work on a Panasonic UCU, but I haven't been able to get it working at all.  So far I have set Tally Arbiter to TSL5, matched the IP address, port, and TSL address that the UCU is set to for IP Tally.  In test mode I'm unable to get tally to light on the UCU.

I'd be curious what you have learned in setting this up.  Is there another step required on the UCU?
